frontend/src/modules/lf/segments/components/view/lf-projects-table.vue

```diff
@@ -6,156 +6,48 @@
     row-key="id"
     :resizable="false"
   >
+    <!-- Status -->
+    <el-table-column width="130" class-name="table-columns !bg-white">
```

Comment:
I think spacing of both the pill and the cells look off. Also not sure about the rounded border of the table. Worth checking
![Screenshot 2024-12-04 at 08 30 18](https://github.com/user-attachments/assets/84941076-abe2-403f-b3bc-b2936f3fdda3)
<img width="621" alt="Screenshot 2024-12-04 at 08 30 27" src="https://github.com/user-attachments/assets/4721231c-3103-489b-8be2-b84a8b88e812">

Reply:
done